src/tests/ftest/erasurecode/ec_offline_rebuild.yaml

```diff
@@ -71,8 +71,9 @@ ior:
   repetitions: 1
   chunk_block_transfer_sizes:
    # [ChunkSize, BlocksSize, TransferSize]
-   - [32M, 128M, 8M]       #Full Striped
-   #- [32M, 128M, 2K]       #Patrtial Striped
+   - [32M, 128M, 8M]       # Full Striped
+   # Remove this line when DAOS-8502 defect is resolved.
+   #- [32M, 16M, 2K]       # Partial Striped
```

Comment:
Is this accidentally got left commented or is supposed to stay commented?

Reply:
It is commented until DAOS-8502 is resolved.